frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -84,106 +95,124 @@ describe('Kubernetes resource CRUD operations', () => {
+          let newContent;
+          // get, update, and set yaml editor content.
+          yamlEditor.getEditorContent().then((content) => {
+            newContent = _.defaultsDeep(
+              {},
+              { metadata: { name, labels: { [testLabel]: testName } } },
+              safeLoad(content),
+            );
+            yamlEditor.setEditorContent(safeDump(newContent, { sortKeys: true })).then(() => {
+              yamlEditor.clickSaveCreateButton();
+              cy.get(errorMessage).should('not.exist');
+            });
           });
         });
-      });
 
-      it('displays detail view for newly created resource instance', () => {
-        cy.url().should('include', `/${name}`);
-        detailsPage.isLoaded();
-        detailsPage.titleShouldContain(name);
-        cy.testA11y(`Details page for ${kind}: ${name}`);
-      });
+        it('displays detail view for newly created resource instance', () => {
+          cy.url().should('include', `/${name}`);
+          detailsPage.isLoaded();
+          detailsPage.titleShouldContain(name);
+          cy.testA11y(`Details page for ${kind}: ${name}`);
+          if (testI18n) {
+            cy.testI18n(
+              [
+                DetailsPageSelector.horizontalNavTabs,
+                DetailsPageSelector.sectionHeadings,
+                DetailsPageSelector.itemLabels,
+              ],
+              ['timestamp'],
+            );
+          }
+        });
 
-      it(`displays a list view for the resource`, () => {
-        cy.visit(
-          `${namespaced ? `/k8s/ns/${testName}` : '/k8s/cluster'}/${resource}?name=${testName}`,
-        );
-        if (namespaced) {
-          // should have a namespace dropdown for namespaced objects');
-          listPage.projectDropdownShouldExist();
-          listPage.projectDropdownShouldContain(testName);
-        } else {
-          // should not have a namespace dropdown for non-namespaced objects');
-          listPage.projectDropdownShouldNotExist();
-        }
-        listPage.rows.shouldBeLoaded();
-        cy.testA11y(`List page for ${kind}: ${name}`);
-      });
+        it(`displays a list view for the resource`, () => {
+          cy.visit(
+            `${namespaced ? `/k8s/ns/${testName}` : '/k8s/cluster'}/${resource}?name=${testName}`,
+          );
+          if (namespaced) {
+            // should have a namespace dropdown for namespaced objects');
+            listPage.projectDropdownShouldExist();
+            listPage.projectDropdownShouldContain(testName);
+          } else {
+            // should not have a namespace dropdown for non-namespaced objects');
+            listPage.projectDropdownShouldNotExist();
+          }
+          listPage.rows.shouldBeLoaded();
+          cy.testA11y(`List page for ${kind}: ${name}`);
+          if (testI18n) {
+            cy.testI18n(
+              [ListPageSelector.tableColumnHeaders],
+              kind !== 'Secret' ? ['yaml-create'] : [],
```

Comment:
Can we add a generic ID for the create button (whether it's YAML or not) so we can remove this special case?

Reply:
fixed.  Had to update in a few places, but isolated to Cypress usage.